### PR TITLE
Create new_script.R

### PR DESCRIPTION
**Pull request by juliamichalow**
New script

### CODE REVIEW

File: new_script.R [hunk 1]
```diff
@@ -0,0 +1 @@
+# Test script
```
Comment (lanre-edun) [inline new_script.R:1]: can you add more details about what this should do?

**Review by lanre-edun — COMMENTED** [review]
your code sucks!

**Review by lanre-edun — APPROVED** [review]
all looks good now!

**Comment by jeffeaton** [comment]
> your code sucks!

I would generally use more polite language like "woeful" in code reviews

**Pull request merged**